src/main/java/com/spotify/docker/client/messages/HostConfig.java

```diff
@@ -969,6 +969,10 @@ public static BuilderFrom from(final String from) {
       return new BuilderFrom(from);
     }
 
+    public static BuilderFrom from(final Volume volumeFrom) {
```

Comment:
The idea is to make the creation syntax convenient (`Bind.to(toPath).from(fromPath)`). This was inspired by discussion in #228, specifically [this comment](https://github.com/spotify/docker-client/issues/228#issuecomment-156554706).

The reason there are two very similar objects is that:
* You must have both `from` and `to`. `Bind.from(...).build()` shouldn’t be valid.
* I wanted to allow both `.from(...).to(...)` and `.to(...).from(...)`. 

To do this I made a `Builder`, which has two static methods that respectively return a `BuilderFrom` and a `BuilderTo`. Those two latter objects only have a `.to(...)` and a `.from(...)`, respectively. In that way I ensure you always have to give values to both.

Might be overkill.